src/selectors/visibleColumnBreakpoints.js

```diff
@@ -112,7 +112,7 @@ export function getColumnBreakpoints(pausePoints, breakpoints, viewport) {
 
     return {
       location,
-      enabled: !!foundBreakpoint,
+      enabled: foundBreakpoint ? !foundBreakpoint.disabled : false,
```

Comment:
would this work?

```js
foundBreakpoint && !foundBreakpoint.disabled
```

Reply:
i wonder why disabled would be undefined... maybe we should change that so that it is true or false?

Reply:
That's what `foundBreakpoint ? !foundBreakpoint.disabled : false` does

Reply:
so if `foundBreakpoint.disabled` were either true or false, `foundBreakpoint && !foundBreakpoint.disabled` would work...

we try our best to keep disabled a boolean in several places

- https://github.com/bomsy/debugger.html/blob/funcition-indexes/src/utils/breakpoint/index.js#L141-L150
- https://github.com/bomsy/debugger.html/blob/funcition-indexes/src/actions/breakpoints/addBreakpoint.js#L77

somehow it must be allowed to become undefined

Reply:
Try executing the following in console:

```js
let foundBreakpoint
foundBreakpoint && !foundBreakpoint.disabled
```

You'll get `undefined`; it's not a bug in our breakpoint code, it's that `foundBreakpoint && !foundBreakpoint.disabled` returns `undefined` because `foundBreakpoint` is `undefined`; I prevent that issue with my ternary:  `enabled: foundBreakpoint ? !foundBreakpoint.disabled : false`

Reply:
got it...  i didn't follow.

Okay, so this is a super nit, but i think i'd prefer `!!foundBreakpoint && !foundBreakpoint.disabled`. Go with whatever looks better to you though.